terraform/cloudfoundry/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "zones" {
```

Comment:
This is duplicated, [also in globals.tf](https://github.com/alphagov/paas-cf/blob/110234556_cf_deploy/terraform/vpc/globals.tf#L15--1)